tftp/tftp.py

```diff
@@ -148,14 +149,20 @@ def __create_thread_and_process_requests(self, pkt, addr):
                                 block_size = int(strings_in_RRQ[index + 1])
 
                         #construct oack
-                        transfer_ack_opcode = pack("!H", TFTPServer.OACK_OPCODE)
-                        oack_data = 'tsize'.encode() + b'\0' + pack("!I", t_size) + b'\0'
-                        oack_data += 'block_size'.encode() + b'\0' + pack("!H", block_size) + b'\0'
-                        packet = transfer_ack_opcode + oack_data
-                        client_dedicated_sock.sendto(packet, addr)
+                        s = serializeme.Serialize({
+                            "opcode"  : ("2B", TFTPServer.OACK_OPCODE),
+                            "option1" : (serializeme.NULL_TERMINATE, "tsize"),
+                            "tsize"   : ("4B", t_size),
+                            "nullterm": "1B",
+                            "option2": (serializeme.NULL_TERMINATE, "block_size"),
+                            "blocksize": ("2B", block_size),
+                            "nullterm2": "1B",
+                        })
+
+                        client_dedicated_sock.sendto(s.packetize(), addr)
 
                     # set the opcode for the packet we are sending
-                    transfer_opcode = pack("!H", TFTPServer.DATA_OPCODE)
+                    # transfer_opcode = pack("!H", TFTPServer.DATA_OPCODE)
```

Comment:
please just remove this line

Reply:
Sorry, didn't catch this! Just removed it with the new commit.